templates/includes/utils.html.twig

```diff
@@ -0,0 +1,10 @@
+{# Twig used as macros holder only #}
+
+{% macro attr_to_html(attr) %}
+    {%- if attr is not empty %}
+        {% for name, value in attr %}
+            {{ name|e('html_attr') }}={{ value|e('html_attr') }}
```

Comment:
are you missing the " here?

Reply:
Then we cannot use it on the attribute name.

But I am pretty sure that the escape filter does not add quotes, but only "fixes wrong character".

Reply:
```suggestion
            {{ name }}={{ value|e('html_attr') }}
```

Reply:
I was 100% certain before that the escaper does not add quotes. 
And now that you linked to the code, I am even more certain :)

    {{ dump('ffooo') }}
    {{ dump('ffo"oo'|e('html_attr')) }}

<img width="131" alt="Bildschirmfoto 2022-05-24 um 15 56 15" src="https://user-images.githubusercontent.com/533162/170053131-a086c764-c9f5-4a14-bdf6-d8693b0dc4b3.png">

Did you check the generated HTML with the developer console, which shows the "tidy html" ?

Reply:
Yes, 
I've already said that in https://github.com/kevinpapst/TablerBundle/issues/85#issuecomment-1135840318

Possible cases:
```html
<div data-good="good boy"></div>
```

```html
<div data-bad-question-mark="I'm not sure what I am doing.."></div>
```

@kevinpapst What did you expect to happen when using RAW HTML like that ?

Reply:
Hey @kevinpapst, back from holidays!

Here is what you want: 

![image](https://user-images.githubusercontent.com/25293190/170955925-57543dcb-b303-4a26-8230-ab6227126fb5.png)


BUT this dump STRINGS not HTML.
I've done the same things for HTML render: 
![image](https://user-images.githubusercontent.com/25293190/170956470-61da4f7d-c022-4df3-b918-1125584332c1.png)

Here's what is looks like in HTML (two last lines):
![image](https://user-images.githubusercontent.com/25293190/170956678-7428f1ac-472f-4305-b5df-a48684df457f.png)

ATE, the render is the same 🤔

Reply:
Back from holidays and directly into such a discussion, I hope it doesn't lower your relaxation level 😁 

It is the same in your browser console, because every browser "fixes" invalid HTML as good as it can. And both the console and the element inspector will NOT show the original HTML which was sent to the browser, but the interpreted/cleaned version of it. 

The first version is not outputting the "" around the attributes value as we can see in the first screenshot. 

According to the RFC both versions are valid: 
https://www.w3.org/TR/2018/SPSD-html5-20180327/syntax.html#attributes-0

> There are four slightly different syntaxes that may be used for attributes in HTML: empty, unquoted, single-quoted and double-quoted. All four syntaxes may be used in the HTML syntax, depending on what is needed for each specific attribute. However, in the XHTML syntax, attribute values must always be quoted using either single or double quotes.

But I would still prefer if we render `Double-quoted attribute value syntax`, which is less error-prone than the `Unquoted attribute value syntax`.

Reply:
Interesting are only:
```
    <div data-good={{ 'good boy' }}></div>
    <div data-good={{ 'good boy'|e('html_attr') }}></div>
```
The first one is wrong, becuase it renders as
    <div data-good=good boy></div> and the browser thinks `data-good=good` is a `Unquoted attribute value syntax` and `boy` is a `Empty attribute syntax`.

Now the same works when you attach the `|e('html_attr')` filter, because the space is escaped and therefor not interpreted as spacer for the next attribute.

Yes, in theory it should not matter if we add " or not, because the filter makes sure that there is no un-escaped space. But in practice it is better to add quotes, to be consistent everywhere.

Do NOT trust your browser console if you want to see the real TWIG output. I had some troublesome debugging sessions because of that misleading behavior! You donÄt see the problem, unless you dump the real HTML before sending it to the browser.